docs/user/aws/install.md

```diff
@@ -60,5 +63,110 @@ The OpenShift console is available via the kubeadmin login provided by the insta
 
 ![OpenShift web console](images/install_console.png)
 
+## Example: User Provided Infrastructure (UPI)
+
+The steps for performing a UPI-based install are outlined here. This example makes use of and reference to several 
+CloudFormation templates located in this repository.
+
+### Create Ignition Configs
+
+The machines will be manually started. Therefore, it is required to generate the bootstrap and machine ignition configs 
+and store them for later steps.
+
+```console
+[~]$ openshift-install-linux-amd64 create ignition-configs
+? SSH Public Key /home/user_id/.ssh/id_rsa.pub
+? Platform aws
+? Region us-east-1
+? Base Domain openshiftcorp.com
+? Cluster Name test
+? Pull Secret [? for help]
+```
+
+After running the command, several files will be available in the directory.
+
+```console
+[~]$ ls
+auth  bootstrap.ign  master.ign  metadata.json  worker.ign
+```
+
+### Create/Identify the VPC to be Used
+
+You may create a VPC with various desirable characteristics for your situation (VPN, route tables, etc.). The recommended
+VPC configuration and a CloudFormation template is provided [here](../../../upi/aws/cloudformation/01_vpc.yaml). 
+
+A created VPC via the template or manually should approximate a setup similar to this:
+
+![Example VPC Architecture Diagram](images/install_upi_vpc.png)
+
+### Create DNS entries and Load Balancers for Control Plane Components
+
+The recommended DNS and load balancer configuration within a CloudFormation template is provided 
+[here](../../../upi/aws/cloudformation/02_cluster_infra.yaml). It uses a public hosted zone and creates a private hosted
+zone similar to the IPI installation method. It also creates load balancers and listeners the same way as the IPI
+installation method. This template can be run multiple times within a single VPC and in combination with the VPC
+template provided above.
+
+#### Create Load Balancer Configuration
+
+It is recommended to create a TCP load balancer for ports 6443 and 22623. The targets will be the master nodes.
+Port 6443 must be accessible to both clients external to the cluster and nodes within the cluster. Port 22623 must
+be accessible to nodes within the cluster. 
+
+#### Create Route53 Hosted Zones & Records
+
+For the cluster name identified earlier in "Create Ignition Configs", you must create a DNS entry which resolves to 
+your created load balancer. The entry api.$clustername.$domain should point to the load balancer.
```

Comment:
Also `*.apps.$clustername.$domain` (which is usually set up by the ingress operator).  Do we want UPI folks to adjust the ingress configuration so the operator doesn't create that route on its own?

Reply:
I want to leave that out ATM. Trying to separate out the UPI portions we give them control over at Day0 and not get into the weeds about the router or the registry at this point. We may need to TBH, but until we document the current state to get a cluster up and then work those mechanics out, it may just be a burden to maintain as this flux.